doc/whats_new/v0.24.rst

```diff
@@ -230,12 +234,15 @@ Changelog
   :meth:`tree.DecisionTreeRegressor.fit`, and has not effect.
   :pr:`17614` by :user:`Juan Carlos Alfaro Jiménez <alfaro96>`.
 
+<<<<<<< HEAD
+=======
 :mod:`sklearn.datasets`
 .......................
 - |Feature| :func:`datasets.fetch_openml` now validates md5checksum of arff
   files downloaded or cached to ensure data integrity.
   :pr:`14800` by :user:`Shashank Singh <shashanksingh28>` and `Joel Nothman`_.
 
```

Comment:
Indeed the entire block should be removed: it is already present on the top.
Modules are in alphabetical order in the what's new file.